apps/web/screens/Article/components/ArticleChatPanel/config.ts

```diff
@@ -12,262 +12,306 @@ export const liveChatIncConfig: Record<string, LiveChatIncChatPanelProps> = {
   },
 }
 
+const setupOneScreenWatsonChatBot = (
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  instance: any,
+  categoryTitle: string,
+  categoryGroup: string,
+) => {
+  if (sessionStorage.getItem(categoryGroup) !== categoryTitle) {
+    sessionStorage.clear()
+  }
+  sessionStorage.setItem(categoryGroup, categoryTitle)
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const preSendhandler = (event: any) => {
```

Comment:
`any` type 😦 

Is it possible to set the correct type?

Reply:
I created an interface above to at least help with the typing a bit